src/components/modals/deactivateCause/WhyContentCause.tsx

```
@@ -0,0 +1,138 @@
+import { AnimatePresence, motion } from 'framer-motion';
+import React from 'react';
+import styled from 'styled-components';
+import { Shadow } from '@/components/styled-components/Shadow';
+import { ISelectObj } from '@/components/modals/deactivateCause/DeactivateCauseIndex';
+import type { CSSObjectWithLabel, StylesConfig } from 'react-select';
+
+interface IWhyContent {
+	handleChange: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+	handleSelect: (e: any) => void;
+	options: ISelectObj[];
+	selectedOption: ISelectObj | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)